crates/bytes-hex/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "bytes-hex"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+serde = { workspace = true }
+serde_json = { workspace = true }
+serde_with = { workspace = true }
+hex = "0.4.3"
```

Comment:
Is there are reason this doesn't also use the workspace `hex` version?

Reply:
Thanks, really good point, I didn't realize it was in the workspace! :+1: